packages/kbn-guided-onboarding/src/types.ts

```diff
@@ -17,15 +17,18 @@ export type GuideId =
 type KubernetesStepIds = 'add_data' | 'view_dashboard' | 'tour_observability';
 type SiemStepIds = 'add_data' | 'rules' | 'alertsCases';
 type SearchStepIds = 'add_data' | 'search_experience';
-type TestGuideIds = 'step1' | 'step2' | 'step3';
+type TestGuideIds = 'step1' | 'step2' | 'step3' | 'step4';
 
 export type GuideStepIds = KubernetesStepIds | SiemStepIds | SearchStepIds | TestGuideIds;
 
+export type GuideParams = Record<string, string>;
+
 export interface GuideState {
   guideId: GuideId;
   status: GuideStatus;
   isActive?: boolean; // Drives the current guide shown in the dropdown panel
   steps: GuideStep[];
+  params?: GuideParams;
```

Comment:
Should we rename this to something more specific? Or do we expect the potential to use this in other ways than just the URL in the future?

Reply:
yes, I thought that we might have other dynamic parameters like integrations or something in the copy. But I can also rename it to something more specific, for example `urlParams` or even just `guideConfigParams` or `guideParams`, wdyt? (naming is hard 😅 )

Reply:
I'm OK to leave as-is for now. We can always revisit.